Firestore/Source/API/FIRFilter+Internal.h

```diff
@@ -93,15 +94,27 @@ NS_SWIFT_NAME(Filter)
                               notIn:(nonnull NSArray<id> *)values
     NS_SWIFT_NAME(whereField(_:notIn:));
 
++ (FIRFilter *)orFilterWithFilters:(NSArray *)filters NS_SWIFT_NAME(orFilter(_:));
+
++ (FIRFilter *)andFilterWithFilters:(NSArray *)filters NS_SWIFT_NAME(andFilter(_:));
+
 @end
 
-/** Internal FIRFilter properties we don't want exposed in our public header files. */
-@interface FIRFilter (Internal)
+/** Exposed internally */
+@interface FSTUnaryFilter : FIRFilter
```

Comment:
`FIR` is used as the prefix of public class and `FST` is the prefix of internal class. Since `FSTUnaryFilter` and `FSTCompositeFilter` is hidden from public, I use `FST` as prefix.